hack/gl-pkg-url.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Parameter 1: garden linux version
+# Parameter 2: ARCH (arm64, amd64, all)
+# Parameter 3: package name (wildcard allowed, but then you must enclouse parameter with quotes)
+# Outputs a list of kernel packages
+#
+# Example: ./gl-pkg-url.sh 934.10 amd64 linux-headers
+
+GL_VERSION=${1:-today}
+GL_ARCH=${2:-"amd64"}
```

Comment:
bit unintuitive to me that amd64 is the default, I would have thought that the current used architecture is the default (i.e. arm on m1 macs)

but I guess this makes sense as most servers probably still run amd64 architecture.

Reply:
True! 
Hm. I don't like the parameter handling I did here. A default does not make too much sense. Would be cleaner to make it mandatory that the user specifies it..